eng/versioning/external_dependencies.txt

```diff
@@ -60,15 +60,20 @@ org.powermock:powermock-api-mockito2;2.0.2
 org.powermock:powermock-module-junit4;2.0.2
 org.slf4j:slf4j-api;1.7.28
 org.slf4j:slf4j-simple;1.7.25
-org.springframework.boot:spring-boot-actuator-autoconfigure;2.2.0.RELEASE
-org.springframework.boot:spring-boot-autoconfigure;2.2.0.RELEASE
-org.springframework.boot:spring-boot-autoconfigure-processor;2.2.0.RELEASE
-org.springframework.boot:spring-boot-configuration-processor;2.2.0.RELEASE
-org.springframework.boot:spring-boot-dependencies;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-test;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-validation;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-web;2.2.0.RELEASE
+
+## Spring boot dependency versions
+org.springframework.boot:spring-boot-actuator-autoconfigure;2.3.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure-processor;2.3.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure;2.3.0.RELEASE
+org.springframework.boot:spring-boot-configuration-processor;2.3.0.RELEASE
+org.springframework.boot:spring-boot-dependencies;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-test;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-validation;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-web;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter;2.3.0.RELEASE
+org.springframework.boot:spring-boot;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-parent;2.3.0.RELEASE
+
```

Comment:
I'm going to mirror what @saragluna mentioned, do we need to update these for security issues? I know that there's a [CVE](https://dev.azure.com/azure-sdk/internal/_componentGovernance/106501/alert/3118418?typeId=1403963) and org.springframework:spring-web:5.2.5.RELEASE is going to need to be updated to a later version.

nit: don't need the extra empty line here.

Reply:
Updated.